packages/utils/src/normalize-path.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Replace windows style backslashes with unix forwards slashes
```

Comment:
Understandably this is loosely based on the previous `normalize-path` package that is removed in this PR, but will we be using this outside of Node environment? 

Wondering can we also use `path.sep`/`path.posix.sep` in Node, similar to their usage in shared util function `pathToRelativeUrl`: 

https://github.com/directus/directus/blob/431f95210a5347a27e50661a2e35c789f161b381/packages/shared/src/utils/node/path-to-relative-url.ts#L4

but also understand that the current `\\`/`/` usages are technically what was already being used in the previous package.

Reply:
> Wondering can we also use path.sep/path.posix.sep in Node, similar to their usage in shared node util function pathToRelativeUrl:

I don't think we can in this case! The goal of the function is to normalize windows style paths no matter what platform you're currently on. It's not trying to make the path "current environment" compatible, the goal is to turn any path into a unix path 🙂

Reply:
Makes sense! Initially this question was typed halfway before I realized it was based on the `normalize-path`, so just wanted to confirm the goal anyways 😄